packages/ses-ava/src/ses-ava-test.js

```diff
@@ -46,45 +66,88 @@ const logErrorFirst = (func, args, name, logger = defaultLogger) => {
   }
 };
 
-const testerMethodsWhitelist = [
+const overrideList = [
   'after',
   'afterEach',
   'before',
   'beforeEach',
   'failing',
   'serial',
   'only',
-  'skip',
 ];
 
 /**
- * @param {TesterFunc} testerFunc
- * @param {Logger} [logger]
- * @returns {TesterFunc} Not yet frozen!
+ * @callback BaseImplFunc
+ * This is the function that invariably starts `t => {`.
+ * Ava's types call this `Implementation`, but that's just too confusing.
+ * @param {Assertions} t
+ * @returns {unknown}
+ *
+ * @typedef {BaseImplFunc | Object} ImplFunc
+ * @property {(...unknown) => string} [title]
+ *
+ * @callback TesterFunc
+ * @param {string} title
+ * @param {ImplFunc} [implFunc]
+ * @returns {void}
  */
-const wrapTester = (testerFunc, logger = defaultLogger) => {
+
+/**
+ * @template {TesterFunc} T
```

Comment:
I think with this we could erase the three custom typedefs above. We shouldn't have to define any types for an Ava wrapper; they should all match Ava's.
```suggestion
 * @template {import('ava').TestFn} T
```

Reply:
Love it! Done.